packages/services/src/view-service/unbonding-tokens-by-address-index/helpers.ts

```diff
@@ -10,10 +10,13 @@ import { getDisplayFromBalancesResponse } from '@penumbra-zone/getters/balances-
 import { status } from '../status';
 import { appParameters } from '../app-parameters';
 
-export const isUnbondingTokenBalance = (balancesResponse: PartialMessage<BalancesResponse>) =>
-  assetPatterns.unbondingToken.matches(
+export const isUnbondingTokenBalance = (balancesResponse: PartialMessage<BalancesResponse>) => {
+  if (balancesResponse.balanceView?.valueView?.case === 'unknownAssetId') return false;
```

Comment:
Perhaps we can make `getDisplayFromBalancesResponse` have an `optional()` on the getter instead?

Reply:
`getDisplayFromBalancesResponse` is not used anywhere else, so yes, it makes sense. Done

Reply:
LGTM 👍